dreem/datasets/sleap_dataset.py

```
@@ -162,12 +159,12 @@ def get_instances(self, label_idx: list[int], frame_idx: list[int]) -> list[Fram
             lf = video[frame_ind]
 
             try:
-                img = vid_reader.get_data(int(lf.frame_idx))
-            except IndexError as e:
-                logger.warning(
-                    f"Could not read frame {frame_ind} from {video_name} due to {e}"
-                )
-                continue
+                img = lf.image
+            except FileNotFoundError as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)